src/ansys/fluent/core/services/health_check.py

```diff
@@ -39,6 +39,8 @@
 
 logger: logging.Logger = logging.getLogger("pyfluent.general")
 
+TIMEOUT = 60
```

Comment:
Please move this to src\ansys\fluent\core\__init__.py as a new configuarion `CHECK_HEALTH_TIMEOUT`. We already have `CHECK_HEALTH` there.

Reply:
Done. Thank you.